Cabal/Distribution/Types/LocalBuildInfo.hs

```diff
@@ -301,3 +308,15 @@ externalPackageDeps lbi =
     -- True if this dependency is an internal one (depends on the library
     -- defined in the same package).
     internal ipkgid = any ((==ipkgid) . componentUnitId) (Graph.toList (componentGraph lbi))
+
+withProfLib :: LocalBuildInfo -> Bool
+withProfLib = isJust . libProfiling
```

Comment:
ide-backpend-0.10.0.1 is using this as a record setter http://hs01.scs.stanford.edu:8910/search/?q=withProfLib+%2B%3D